assets/src/stories-editor/components/font-family-picker/index.js

```diff
@@ -6,13 +6,19 @@ import PropTypes from 'prop-types';
 /**
  * WordPress dependencies
  */
-import { __, sprintf } from '@wordpress/i18n';
-
+import { __, _n, sprintf } from '@wordpress/i18n';
+import { BaseControl } from '@wordpress/components';
+/**
+ * Internal dependencies
```

Comment:
Thee grouping of the dependencies in this file got a bit mixed up / duplicated

Reply:
Resolved in https://github.com/ampproject/amp-wp/pull/3155/commits/f958f54816f234df0c749998e06e3931682abf31